public/sandbox-data/apprenticeship-levy/employers/257LLJCAP2INT/employed/AA567890C.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "jsonBody": {
+    "empref": "257/LLJCAP2INT",
+    "nino": "AA567890C",
```

Comment:
Nino

Reply:
Ill address this in a later ticket. This is a bigger piece of work as App-levy is using itself as a stub. We need to create a separate microservice for this.